services/user.py

```diff
@@ -0,0 +1,39 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
```

Comment:
Consider replacing the direct import of `User` with `get_user_model()` to ensure compatibility with custom user models.